packages/common/src/dto/custom-decorator.ts

```diff
@@ -0,0 +1,20 @@
+import { registerDecorator, ValidationArguments, Validator } from 'class-validator';
+import { FormClass } from './form-class.dto';
+
+const validator = new Validator();
+export function ValidateNestedObject() {
```

Comment:
is `custom-decorator` the right name for this file? It's written specifically for the Form class so something like `FormValidator` might be more appropritate